### DISCUSSION TITLE
Alertmanager skip TLS verification

### DISCUSSION THREAD

**Discussion by praneethkaturi** [jertel/elastalert2#869] [Q&A] [answered]
How do I skip tls verification when posting the alert to alertmanager? 
I see these two options :- 
```
alertmanager_ca_certs: False
alertmanager_ignore_ssl_errors: False
```

And I've set both of them to false, I still get the following error:-
`ERROR:elastalert:Error while running alert alertmanager: Error posting to Alertmanager: HTTPSConnectionPool(host='***', port=9094): Max retries exceeded with url: /api/v1/alerts (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self signed certificate in certificate chain (_ssl.c:997)')))`

But when I curl it using the -k option, I do get a valid response

**Comment by nsano-rururu** [comment, marked as answer]
current

```yaml
alertmanager_ca_certs: False
alertmanager_ignore_ssl_errors: False
```

fix

```yaml
alertmanager_ignore_ssl_errors: True
```

**Reply by praneethkaturi** [reply]
According to the official docs: https://elastalert2.readthedocs.io/en/latest/ruletypes.html

`alertmanager_ignore_ssl_errors: By default ElastAlert 2 will verify SSL certificate. Set this option to False if you want to ignore SSL errors.`

**Reply by nsano-rururu** [reply]
@praneethkaturi 

How was the result?
If the documentation is incomplete, you can just make a pull request

**Reply by jertel** [reply]
I've corrected the documentation.

**Reply by nsano-rururu** [reply]
The following documents have similar deficiencies.

http_post_ignore_ssl_errors
http_post2_ignore_ssl_errors
mattermost_ignore_ssl_errors
ms_teams_ignore_ssl_errors
rocket_chat_ignore_ssl_errors
slack_ignore_ssl_errors

**Reply by jertel** [reply]
Fixed.

**Reply by praneethkaturi** [reply]
Thank you for changing the documentation, it works now!
